ios/RNSentry.h

```diff
@@ -32,3 +32,15 @@ SentrySDK (Private)
                                        symbolicate: (SymbolicateCallbackType) symbolicate;
 
 @end
+
+@interface SentryCurrentDateProvider : NSObject
+
+- (NSDate *)date;
+
+- (dispatch_time_t)dispatchTimeNow;
+
+- (NSInteger)timezoneOffset;
+
+- (uint64_t)systemTime;
```

Comment:
l: If you're not using all the functions you can remove it from here to keep this short.